lib/components/narrative/metro/departure-times-list.tsx

```diff
@@ -58,10 +54,7 @@ export const DepartureTimesList = (props: DepartureTimesProps): JSX.Element => {
             className={classNames.join(' ')}
             key={getFirstLegStartTime(time.legs)}
             onClick={() => setItineraryTimeIndex(index)}
-            title={`${intl.formatMessage(
-              { id: 'components.MetroUI.arriveAtTime' },
-              { time: intl.formatTime(getLastLegEndTime(time.legs)) }
-            )} ${
+            title={`${intl.formatTime(getFirstLegStartTime(time.legs))} ${
```

Comment:
Based on our discussion I would remove this title as well. It only shows the time for me and a space.